app/src/main/java/sk/inloop/support/v4/app/FragmentStatePagerAdapter.java

```diff
@@ -0,0 +1,248 @@
+package sk.inloop.support.v4.app;
+
+import android.os.Bundle;
+import android.os.Parcelable;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v4.view.PagerAdapter;
+import android.util.Log;
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by adammihalik on 05/12/2016.
+ */
+
+public abstract class FragmentStatePagerAdapter extends PagerAdapter {
+
+    private static final String TAG = FragmentStatePagerAdapter.class.getSimpleName();
+    private static final boolean DEBUG = false;
+
+    private final FragmentManager mFragmentManager;
```

Comment:
Please use @NonNull / @Nullable for every field, every parameter and every return value.